plugins/cookie_manager/test/cookies_test.dart

```diff
@@ -139,6 +139,41 @@ void main() {
       expect(response.requestOptions.headers[HttpHeaders.cookieHeader], null);
     });
   });
+
+  test('cookies replacement', () async {
+    final cookies = [
+      Cookie('foo', 'bar'),
+      Cookie('a', 'c'),
+    ];
+
+    final previousCookies = [
+      Cookie('foo', 'oldbar'),
+      Cookie('d', 'e'),
+      Cookie('e', 'f'),
+    ];
+
+    final newCookies = CookieManager.getCookies(
+      [...cookies, ...previousCookies],
+    );
+
+    expect(newCookies, 'foo=bar; a=c; d=e; e=f');
+  });
+
+  test('testing cookies RFC6265 5.4 #2 sorting', () async {
```

Comment:
A test named "test" is inefficient.

```suggestion
  test('RFC6265 5.4 #2 sorting', () async {
```